corehq/apps/accounting/static/accounting/js/credits.js

```diff
@@ -128,6 +131,17 @@ hqDefine('accounting/js/credits', [
         return self;
     };
 
+    $(function () {
+        $('#paymentModal').on('shown.bs.modal', function () {
+            $("#prepay-date-start").datetimepicker({
```

Comment:
can these use any existing knockout bindings? not a fan of having this format manually defined, but maybe I'm thinking of bootstrap 5 page support

Reply:
I think that might be B5. The places I saw `datetimepicker` used were either essentially this same thing ([here ](https://github.com/dimagi/commcare-hq/blob/de22ac74cabe41e538812da5d5d6f8967424d1b9/corehq/apps/users/static/users/js/mobile_workers.js#L545) and [here](https://github.com/dimagi/commcare-hq/blob/de22ac74cabe41e538812da5d5d6f8967424d1b9/corehq/apps/users/static/users/js/edit_commcare_user.js#L141) ) or specific to web apps.